src/components/PRList.tsx

```
@@ -0,0 +1,195 @@
+        </div>
+      </div>
+
+      {/* Filters and Actions */}
+      <div className="card bg-base-100 shadow-md mb-6">
+        <div className="card-body">
+          <div className="flex flex-wrap gap-4">
+            <div className="form-control flex-1 min-w-[200px]">
+              <label className="label">
+                <span className="label-text">Filtrar por repositório</span>
+              </label>
+              <input
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)